synfig-studio/src/gui/resources/ui/Makefile.am

```diff
@@ -6,7 +6,8 @@ UIFILES     = \
   canvas_options.glade \
   preview_options.glade \
   dialog_workspaces.glade \
-  shortcuts_window.glade
+  shortcuts_window.glade \
+	vectorizer_settings.glade
```

Comment:
Two spaces indentation used for legacy reasons.

You used tab character. Change it to 2 spaces for consistency.